code/modules/projectiles/gun_attachables.dm

```diff
@@ -629,7 +635,7 @@ Defined in conflicts.dm of the #defines folder.
 	recoil_mod = -RECOIL_AMOUNT_TIER_4
 	accuracy_unwielded_mod = HIT_ACCURACY_MULT_TIER_4
 	recoil_unwielded_mod = -RECOIL_AMOUNT_TIER_4
-	damage_mod = BULLET_DAMAGE_MULT_TIER_1
+	damage_mod = BULLET_DAMAGE_MULT_TIER_2
```

Comment:
this attachment is not supposed to give + dmg , from what i am aware the fact that it gives a DMG boost is an oversight. 
RC downside is supposed to be the -10% dmg but this one is only positive buffs to the weapon , i think this should be considered.

Reply:
the problem is that RC downside is suposed to be the damage reduction , it makes no sense to have attachments that are ++ versions